css/base.css

```diff
@@ -177,6 +177,11 @@ input:not([type]), select {
     width: 100%;
 }
 
+input[type=text] {
+    color: inherit;
+    font: inherit;
+}
```

Comment:
This could have wider effects than intended, but no matter how much I tried to modify the themes in `_codemirror.ts` the `:` input box got browser-default styles... I can keep investigating that, but this seems to work in the meantime.

Reply:
At the very least move this rule to hole.css then so it wont affect other forms on the site?

Reply:
Could also scope it better? Maybe something like `.cm input[type=text]`, not sure if `cm` is the right class name mind.

Reply:
I moved it to `hole.css` and verified the selector I used (`.cm input`) only selected the relevant input field.